src/node_builtins.cc

```diff
@@ -90,15 +90,23 @@ void BuiltinLoader::GetNatives(Local<Name> property,
   auto source = env->builtin_loader()->source_.read();
   for (auto const& x : *source) {
     Local<String> key = OneByteString(isolate, x.first);
-    if (out->Set(context, key, x.second.ToStringChecked(isolate)).IsNothing()) {
+    Local<String> value;
+    if (!x.second.ToString(isolate).ToLocal(&value)) {
+      return;
+    }
+    if (out->Set(context, key, value).IsNothing()) {
       return;
     }
   }
```

Comment:
Just a suggestion, but if we add support for `UnionBytes` in `ToV8Value()`, this entirely loop could be replaced by a `ToV8Value()` call